examples/create-account-authorization.js

```diff
@@ -40,6 +44,18 @@ async function main () {
     { type: 'WALLET', privateKey: privKey1 }
   )
 
+  // WAIT until account is created
+  var pollingAccountCreate = true
+  while (pollingAccountCreate) {
```

Comment:
Carlos added some helpers that might be useful for this https://github.com/hermeznetwork/hermezjs/blob/develop/tests/helpers/helpers.js#L38

Reply:
I understand, but i dont want to add any dependency on external libraries for the examples.